virtcontainers/kata_agent_test.go

```diff
@@ -248,6 +248,10 @@ func (p *gRPCProxy) GetGuestDetails(ctx context.Context, req *pb.GuestDetailsReq
 	return &pb.GuestDetailsResponse{}, nil
 }
 
+func (p *gRPCProxy) MemHotplugByProbe(ctx context.Context, req *pb.MemHotplugByProbeRequest) (*gpb.Empty, error) {
+	return &gpb.Empty{}, nil
+}
+
```

Comment:
@zhabinecho I think something went wrong with the rebase of master branch here. Those changes have not been introduced by your PR, right?

Reply:
Sorry, I should add one comment to describe it. The agent also add MemHotplugByProbeRequest, StartTracingRequest and StopTracingRequest. The continuous-integration/travis-ci will failed:

virtcontainers/kata_agent_test.go:262:36: cannot use g (variable of type *gRPCProxy) as github.com/kata-containers/runtime/vendor/github.com/kata-containers/agent/protocols/grpc.AgentServiceServer value in argument to pb.RegisterAgentServiceServer: missing method MemHotplugByProbe (typecheck)

		pb.RegisterAgentServiceServer(s, g)

Also the StartTracingRequest and StopTracingRequest are the same.

Reply:
Oh I see, those have not been updated from a runtime perspective yet... Then that's fine, let's approve this PR :)